StreamChatUITestsAppUITests/Tests/StreamTestCase.swift

```diff
@@ -40,6 +40,16 @@ class StreamTestCase: XCTestCase {
 }
 
 extension StreamTestCase {
+    
+    enum Tags: String {
+        case general
```

Comment:
Maybe something like this? :) 

```suggestion
        case coreFeature
```